### PR TITLE
Add test with jira issue id in docstr

### PR DESCRIPTION
**Pull request by vkondula**
Related to #34

### CODE REVIEW

File: tests/test_jira.py [hunk 1]
```diff
@@ -316,3 +316,20 @@ def test_fail():
...
     assert_outcomes(result, 0, 0, 0, xfailed=1)
+
+
+def test_docstr_marker_xfail(testdir):
+    '''docstring with jira issue id'''
+    testdir.makeconftest(CONFTEST)
+    testdir.makepyfile("""
+        import pytest
+
+        @pytest.mark.jira("ORG-1412", run=True)
```
Comment (lukas-bednar) [inline tests/test_jira.py]: Topic says that "Add test with jira issue id in docstr", but I can see using both: decorator, and doc-string as well.

Comment (vkondula) [inline tests/test_jira.py]: > but I can see using both: decorator, and doc-string
That's how I understood "Write test where we get ID from decorator and from doc string"  (#34). It was skipped due to JIRA ID in doc-string. That's why the name. Do you prefer to rename the commit or rewrite the test?

Comment (lukas-bednar) [inline tests/test_jira.py]: yes, please :smile:

**Comment by lukas-bednar** [comment]
@vkondula it will require rebase and resolve conflicts :cry:

**Comment by vkondula** [comment]
@lukas-bednar or you can close it. The issue #34 was resolved in #44 by "test_strategy_strict_exception" and "test_issue_not_found_considered_open_xfailed"
Therefore there is no need for this PR

**Comment by lukas-bednar** [comment]
@vkondula even better, good. resolved via #44

**Pull request closed without merging**